ts/features/itwallet/common/utils/itwSentryUtils.ts

```diff
@@ -0,0 +1,25 @@
+import * as Sentry from "@sentry/react-native";
+
+/**
+ * Sends an exception to Sentry with additional context and tags.
+ * If sendRequired is true, the exception will be sent regardless of any sampling logic.
+ * @param {unknown} exception - The exception to be sent to Sentry.
+ * @param {string} [section] - Optional section of the application where the error occurred.
+ * @param {boolean} [sendRequired] - Indicates whether sending the exception is required.
+ */
+export function itwSendExceptionToSentry(
+  exception: unknown,
+  section?: string,
+  sendRequired?: boolean
+) {
```

Comment:
This function can be useful for every context. We could move it to a common folder
Also, `required` is shorter and still has a lot of context of what it does. What do you think?
```suggestion
export function sendExceptionToSentry(
  exception: unknown,
  section?: string,
  required?: boolean
) {
```